src/main/java/Utils/messageHandler.java

```diff
@@ -0,0 +1,150 @@
+            String doneSign = "";
+            if(isDone) {
+                doneSign = "X";
+            }
+            else {
+                doneSign = "";
+            }
+            type = messages.get(i-1).getType();
+            String typeSign = "";
+            String startTime = "";
+            String endTime = "";
+            String By = "";
+            String From = "";
+            if(type == 1){
+                typeSign = "[T]";
+            }
+            else if(type == 2){
+                By = " By: ";
+                typeSign = "[D]";
+                endTime = messages.get(i-1).getEndTime();
+            }
+            else if(type == 3){
+                typeSign = "[E]";
+                By = " By: ";
+                From = " From: ";
+                startTime = messages.get(i-1).getStartTime();
+                endTime = messages.get(i-1).getEndTime();
+            }
+            System.out.println(i + ". " + typeSign + "[" + doneSign + "] " + messages.get(i-1).getMessage() + From + startTime + By + endTime );
+            i++;
+        }
+        System.out.println("-----------------------------------\n");
+    }
+
+    public static void addList(messageList list, String input){
+        if(input.contains("todo")) {
+            List<Message> messages = list.getMessages();
+            Message message = new Message(input);
+            String[] strings = input.split(" ");
+            String eventName = strings[1].split(" ")[1];
+            messages.add(message);
+            list.setMessages(messages);
+            System.out.println("-----------------------------------");
+            System.out.println("added:" + message.getMessage());
+        }
+
+        else if(input.contains("deadline")) {
+            List<Message> messages = list.getMessages();
+            String[] strings = input.split("/");
+            String endTime = strings[strings.length-1].split(" ")[1];
+            String eventName = strings[strings.length-2].split(" ")[1];
+            Message message = new Message(eventName, endTime, 2);
+            messages.add(message);
+            list.setMessages(messages);
+            System.out.println("-----------------------------------");
+            System.out.println("added:" + message.getMessage());
+        }
+
+        else if(input.contains("event")) {
+            List<Message> messages = list.getMessages();
+            String[] strings = input.split("/");
+            String startTime = strings[strings.length-2].split(" ")[1];
+            String endTime = strings[strings.length-1].split(" ")[1];
+            String eventName = strings[strings.length-3].split(" ")[1];
+            Message message = new Message(eventName, startTime, endTime, 3);
+            messages.add(message);
+            list.setMessages(messages);
+            System.out.println("-----------------------------------");
+            System.out.println("added:" + message.getMessage());
```

Comment:
Perhaps you could refactor these 3 parts into methods to make this part more concise and readable.